tests_practice/unittests.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+
+from tests_practice.definitions import (
+    task1_common_elements,
+    task2_letter_times,
+    task3_is_power,
+    task4_add_repeatedly,
+    task5_push_zeros,
+    task6_arithmetic_progression_check,
+    task7_unique_number,
+    task8_missing_number,
+    task9_find_before_tuple,
+    task10_string_reversed
+)
+
+
+class TestCases(unittest.TestCase):
+    """
+    ======== here comes unittests for tasks 1-10 ========
+    """
+
+    def test_task1_common_elements(self):
+        input_list = [
+            [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89],
+            [1, 1, 5, 3, 2, 13, 8, 21, 34, 55, 89]
+        ]
+        check_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+        result = [1, 2, 3, 5, 8, 13]
+        for elem in input_list:
+            with self.subTest(elem=elem):
```

Comment:
```suggestion
            with self.subTest(i=elem):
```
інакше не запуститься сабтест

Reply:
ну і в наступних тестах аналогічно